CPPLINT.cfg

```diff
@@ -1,2 +1,6 @@
 set noparent
 filter=-build/include_dirname,-build/c++11
+filter=-build/include_subdir
+filter=-build/include_order
+filter=-build/include_what_you_use
+filter=-runtime/string
```

Comment:
I had to add these filters after updating cpplint.py to a newer version.

Reply:
I think it would have been nice to update cpplint.py in a separate PR to keep the diff small and targeted
That being said, if you do it as part of this PR, I think we should try to address these warnings as much as possible instead of silencing them (and since this requires changing C++ code potentially, I think that's another argument in favor of doing it in a separate PR).

Reply:
Unfortunately, `2to3` does not really work well on `cpplint.py`. The changes necessary are a lot more involved. So I thought it would be best to just use a recent version.

Maybe the C++ changes can be addressed in a separate PR? At least `runtime/string`, `build/include_what_you_use`, and `build/include_what_you_use` can be done quickly. `build/include_subdir` corresponds to the old `build/include_dirname` I believe.

Reply:
we could have left it for a future PR actually. There was no need to update cpplint.py to work with Python3 at the same time as the rest of the Python code.

Reply:
Reverted the changes. cpplint.py is now run with Python 2 again.